sim/priest/mind_blast.go

```diff
@@ -10,19 +10,42 @@ import (
 func (priest *Priest) registerMindBlastSpell() {
 	baseCost := priest.BaseMana() * 0.17
 
-	Mult_mod := (1 + float64(priest.Talents.Darkness)*0.02) *
+	effect := core.SpellEffect{
+		ProcMask:             core.ProcMaskSpellDamage,
+		BonusSpellHitRating:  0 + float64(priest.Talents.ShadowFocus)*1*core.SpellHitRatingPerHitChance,
+		BonusSpellCritRating: float64(priest.Talents.MindMelt) * 2 * core.CritRatingPerCritChance,
+		DamageMultiplier:     1,
+		ThreatMultiplier:     1 - 0.08*float64(priest.Talents.ShadowAffinity),
+		OutcomeApplier:       priest.OutcomeFuncMagicHitAndCrit(priest.SpellCritMultiplier(1, float64(priest.Talents.ShadowPower)/5)),
+	}
+
+	normalCalc := core.BaseDamageFuncMagic(997, 1053, 0.429)
+	miseryCalc := core.BaseDamageFuncMagic(997, 1053, (1+float64(priest.Talents.Misery)*0.05)*0.429)
+
+	normMod := (1 + float64(priest.Talents.Darkness)*0.02) * // initialize modifier
+		core.TernaryFloat64(priest.Talents.Shadowform, 1.15, 1) *
```

Comment:
we talked in discord already, but we should move this static bonus to talents.go as a bonus to ShadowDamageDealt